### PR TITLE
Update ui for text overflow

### PR DESCRIPTION
**Pull request by nhamhung**
1. Bring back the scroll bars
2. Text overflow will print "..."

### CODE REVIEW

File: src/main/resources/view/DarkTheme.css [hunk 1]
```diff
@@ -345,6 +346,10 @@
     -fx-padding: 3 1 3 1;
 }
 
+.scroll-bar {
```
Comment (helloImHai) [inline src/main/resources/view/DarkTheme.css:349]: LOL WHAT THE =))))

File: src/main/resources/view/DarkTheme.css [hunk 2]
```diff
@@ -345,6 +346,10 @@
...
+
+}
+
```
Comment (helloImHai) [inline src/main/resources/view/DarkTheme.css]: LOL WHAT THEE NQH

**Review by helloImHai — CHANGES_REQUESTED** [review]
Please play around with the UI more when you make changes, I like some of the changes but it breaks things.
Screenshot 2020-04-13 at 12 12 49 AM

**Review by helloImHai — APPROVED** [review]
As it should be m8 💯💯💯

File: src/main/resources/view/DarkTheme.css (hunk 2 above)
Comment (nhamhung) [reply src/main/resources/view/DarkTheme.css]: Sorry~~

**Pull request merged**